README.md

```diff
@@ -219,6 +220,24 @@ Section containing content and actions on the same topic.
 | `minHeight`  | `string` | `auto` | - | CSS minHeight property |
 | `children`  | `JSX.Element` | - | - | Any content that is internally wrapped in a Box |
 
+### Checkbox
+
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/stories/Input.js)
+
+#### Props
```

Comment:
Maybe you can just link to the grommet documentation for anything that is not rendition-specific. Like this we avoid unnecessary duplication.